src/ol/sphere/sphere.js

```diff
@@ -58,19 +57,19 @@ ol.Sphere.prototype.cosineDistance = function(c1, c2) {
 
 
 /**
- * Returns the geodesic area for a list of coordinates.
- *
- * [Reference](http://trs-new.jpl.nasa.gov/dspace/handle/2014/40409)
- * Robert. G. Chamberlain and William H. Duquette, "Some Algorithms for
- * Polygons on a Sphere", JPL Publication 07-03, Jet Propulsion
- * Laboratory, Pasadena, CA, June 2007
- *
- * @param {Array.<ol.Coordinate>} coordinates List of coordinates of a linear
- * ring. If the ring is oriented clockwise, the area will be positive,
- * otherwise it will be negative.
- * @return {number} Area.
- * @api
- */
+* Returns the geodesic area for a list of coordinates.
+*
+* [Reference](http://trs-new.jpl.nasa.gov/dspace/handle/2014/40409)
+* Robert. G. Chamberlain and William H. Duquette, "Some Algorithms for
+* Polygons on a Sphere", JPL Publication 07-03, Jet Propulsion
+* Laboratory, Pasadena, CA, June 2007
+*
+* @param {Array.<ol.Coordinate>} coordinates List of coordinates of a linear
+* ring. If the ring is oriented clockwise, the area will be positive,
+* otherwise it will be negative.
+* @return {number} Area.
+* @api
+*/
```

Comment:
Thanks for the additional detail.  I'd recommend against using `fixjsstyle.py`.  And please ignore that the current lint task asks you to do so.  I don't know if any other developers use this, but clearly it is not doing the right thing.